src/main/java/cs/ualberta/cmput402/tictactoe/TicTacToeGame.java

```diff
@@ -29,10 +35,29 @@ public void promptNextPlayer(){
         }
     }
 
+    public void printScoreBoard(){
+        System.out.println("\n-------------------------------------------");
+        System.out.println("S C O R E B O A R D");
+        System.out.println("-------------------------------------------");
+        System.out.println(String.format("X Wins: %d", this.playerXWins));
+        System.out.println(String.format("O Wins: %d", this.playerOWins));
+        System.out.println(String.format("Ties: %d", this.ties));
+        System.out.println("-------------------------------------------");
+    }
+
+    public void resetGame(){
+        board = new Board();
+    }
+
     public void playGame(){
         Scanner keyboardScanner = new Scanner(System.in);
 
         while (board.getWinner() == null){
+        	
+            if(board.isFull()) {
+            	break;
```

Comment:
* Here is an inconsistent use of tab and space. This tab should be replaced by spaces

```suggestion
                break;
```